vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -201,6 +205,10 @@ module Tuples =
     let tuple2 = (1, "fred", 3.1415)
 
     printfn "tuple1: %A\ttuple2: %A" tuple1 tuple2
+    
+    /// A simple tuple of integers with a type annotation.
+    /// Type annotations for tuples use the * symbol to separate elements
+    let tuple3: (int*int) = (5, 9)
```

Comment:
We normally do not surround type annotations with parens, and not always place parens around tuple values, so `let tuple3 : int * int = 5, 9` is a more idiomatic code.